modules/persistent-storage-csi-migration-sc.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-implications_{context}"]
+= Storage class implications
+
+For new {product-title} 4.13, and later, installations, the default storage class is the CSI storage class. All volumes provisioned using this storage class are CSI persistent volumes (PVs).
```

Comment:
New installations or newly provisioned volumes?

Reply:
new installation (clusters)

Reply:
@sferich888 My understanding is that this is for new installations, and thus any newly provisioned volumes after that new installation. I'm not sure I understand your question.